docs/source/conf.py

```diff
@@ -3315,6 +3315,8 @@
     "ConcatDataset",
     # torch.utils.data.distributed
     "DistributedSampler",
+    # torch.utils.data.distributed
+    "DistributedWeightedRandomSampler",
```

Comment:
Adding new entries in this file is not ok.
Please make sure this is properly documented on the website!